sql/server_component/mysql_thd_attributes_imp.cc

```diff
@@ -50,6 +50,14 @@ DEFINE_BOOL_METHOD(mysql_thd_attributes_imp::get,
         *((bool *)inout_pvalue) = t->is_server_upgrade_thread();
       } else if (!strcmp(name, "is_init_file_thread")) {
         *((bool *)inout_pvalue) = t->is_init_file_system_thread();
+      } else if (!strcmp(name, "m_db")) {
+        *((LEX_CSTRING *)inout_pvalue) = t->db();
+      } else if (!strcmp(name, "m_examined_row_count")) {
+        *((ha_rows *)inout_pvalue) = t->get_examined_row_count();
```

Comment:
ha_rows is an internal type. Please use one of the C types that's closest. e.g. an unsigned long long.